src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import './HomePage.css'
+
+function HomePage() {
+  const [drinkType, setDrinkType] = useState('')
+  const navigate = useNavigate()
+
+  const handleChoice = (choice) => {
+    if (choice === 'alcoholic') {
+      navigate('/all')
```

Comment:
According to the [router docs](https://reactrouter.com/en/main/hooks/use-navigate), it's better to use `redirect` in cases like this.